src/uwtools/resources/jsonschema/ww3.jsonschema

```diff
@@ -0,0 +1,32 @@
+{
+  "properties": {
+    "ww3": {
+      "additionalProperties": false,
+      "properties": {
+        "namelist": {
+          "additionalProperties": false,
+          "properties": {
+            "template_file": {
+              "type": "string"
+            },
+            "template_values": {
+              "$ref": "urn:uwtools:namelist"
```

Comment:
```suggestion
              "additionalProperties": {
                "type": "string"
              },
              "type": "object"
```
I think... We cannot reference the `namelist.jsonschema` here, because it describes a 2-level data structure (namelists -> variables -> values), and for template rendering we want a 1-level structure (keys -> values).

Reply:
Mm, and even then we probably want a clean break at all points, now that I think about it